### DISCUSSION TITLE
Radix UI Sidebar

### DISCUSSION THREAD

**Discussion by abskrj** [radix-ui/themes#85] [General]
Want to use the RadixUI Sidebar used on the website iteself. Is there a way we can use that?

**Comment by kevinmitch14** [comment]
The website is open source [here](https://github.com/radix-ui/website).


[components/SideNav.tsx](https://github.com/radix-ui/website/blob/main/components/SideNav.tsx#L3) might be a good place to start

**Comment by abskrj** [comment]
Thanks @kevinmitch14
